src/components/Typography/Headline.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { ReactNode } from "react";
+import cx from "classnames";
+import { prefix } from "../../settings";
+
+type HeadlineProps = {
+  /**
+   * Aspect Ratio Children
+   */
+  children?: ReactNode;
+
+  /**
+   * Type to use
+   */
+  type?: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+
+  /**
+   * Classnames
+   */
+  className?: string;
+
+  /**
+   * Title
+   */
+  title?: string;
+
+  /**
+   * React inline styles for the typography component
+   */
+  style?: any;
+
+  /**
+   * HTML for label
+   */
+  htmlFor?: string;
+
+  name?: string;
+};
+
+const Headline = ({ children, className, type, ...rest }: HeadlineProps) => {
+  return (
```

Comment:
```suggestion
  const Element = React.createElement(type);
  return (
```
und dann unten einfach <Element> verwenden.

Reply:
Als `<Element>`